tests/test65.c

```diff
@@ -16,19 +16,6 @@ struct CacheEntry {
 };
 struct CacheEntry *cache = NULL;
 
-static char * /*value*/ find_in_cache(const char *key)
```

Comment:
Well, then, a bit more verbosity on those printtables isn't going to be a problem either :)